source/stylesheets/layout/_form.scss

```diff
@@ -0,0 +1,46 @@
+.form-box {
+  background-color: $turquoise;
+  height: 300px;
+  width: 100%;
+}
+
+.form-box-content {
+  h2 {
+    color: $white;
+    text-align: center;
+    font-weight: 300;
+    letter-spacing: 1px;
+    text-transform: uppercase;
+    padding-bottom: 10px;
+  }
+}
+
+.form-column {
+
+  input {
+    border: 0;
+    border-radius: 7px;
+    box-shadow: none;
+    clear: both;
+    display: block;
+    margin: 7.5px 5px;
+    padding: 5px 10px;   
+    width: 285px;  
+  }
+
+  .input-message {
+    height: 120px;
+    padding-bottom: 90px;
+  }
+}
+
+.send {
+  float: left; 
+  font-size: $small-font;
+  margin: 2 * $base-margin auto;
+  padding: 12px 35px;
+}
+
+.icon-mail {
+  margin-right: 2 * $base-margin;
```

Comment:
It is not connected with icon and mail. It only ests margin. Must be more generic ```push-double-right``` should be moved to /utilis.scss